### ISSUE TITLE
Site https://vicopo.selfbuild.fr/ down

### ISSUE THREAD

**Issue by fabinou** [kylekatarnls/vicopo#16]
Bonjour,

Tout d'abord merci pour votre travail et cette excellente API.

Malheureusement, depuis ce matin j'ai l'impression que le site est HS (timeout). Est ce du à une éventuelle maintenance ?

Merci encore,

**Comment by kylekatarnls** [comment]
Bonjour,

En effet, il est tombé à 11h17, j'ai informé mon hébergement dans la foulée. Je ne peux rien faire de plus pour le moment.

Merci de votre patience.

**Comment by fabinou** [comment]
Bonjour,

Merci infiniment pour ces informations.

Bon courage !

**Comment by yoloSwagSEO** [comment]
Merci!

**Comment by fabinou** [comment]
Bonjour

Si vous voulez je peux vous  proposer un hosting de secours ?

**Comment by kylekatarnls** [comment]
Oui, mon hosting m'a assuré rétablir la situation dans la journée, si à 16h (GMT+01:00), ce n'est pas rétabli, je déploierai un nouvel hébergement, le délais sera alors ensuite la propagation DNS.

Dans les deux cas, je vais intégrer dans une prochaine version une URL de fallback directement dans la librairie.

Je rappelle à toute fin utile que VICOPO est fourni sans garantie, ne génère aucun revenu, est hébergé à mes frais, développé et maintenu sur mon temps libre.

Donc malgré les millions de requêtes que ça représente et quel que soit les implications finales, ça ne peut pas être ma priorité.

Merci de votre compréhension.

**Comment by fabinou** [comment]
Merci pour ces infos.

Et pour moi  il est clair que c'est un service gracieux sans SLA, donc n'hésitez pas si vous avez besoin d'un hosting de secours pour vous aider

**Comment by yoloSwagSEO** [comment]
> Bonjour,
> 
> Avez-vous un délais de rétablissement à nous communiquer ? ou une solution de hosting de secours car de nombreuses applications dont nous avons TMA à charge utilise cette api et cela deviens problématique.
> 
> Merci d'avance.

Normalement on utilise pas un service sans SLA en production ^^

**Comment by peninonwilliam** [comment]
> > Bonjour,
> > Avez-vous un délais de rétablissement à nous communiquer ? ou une solution de hosting de secours car de nombreuses applications dont nous avons TMA à charge utilise cette api et cela deviens problématique.
> > Merci d'avance.
> 
> Normalement on utilise pas un service sans SLA en production ^^

Suis d'accord avec toi mais va dire ça aux clients ! quand le client te dit on veut utiliser ce truc la !
